test.sh

```diff
@@ -502,12 +502,70 @@ function test_docs {
   diff docs/reference/config.yaml $kb_orig/test/docs/expected/config.yaml
 }
 
+function generate_controller {
+  header_text "creating controller"
+    kubebuilder create controller --group ant --version v1beta1 --kind Ant
+}
+
+function update_controller_test {
+  # Update import
```

Comment:
Its not very clear what are we doing here ?

Reply:
Explained offline. Also added a comment for explanation.